problem_builder/translations/ar/LC_MESSAGES/text.po

```diff
@@ -2,216 +2,211 @@
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
+# 
+# Translators:
+# 6e68c7971a89e50e680ae9444d303c8f, 2019
+# Ahmad AbdArrahman <mygooglizer@gmail.com>, 2019
+# Najwan Al Rousan <najwanrousan@gmail.com>, 2019
+# Nabeel El-Dughailib <nabeel@qordoba.com>, 2019
+# may <may@qordoba.com>, 2019
+# Ahmed Jazzar <ahmed.mojaz@gmail.com>, 2019
+# Soha Assali <soha+transifex@qordoba.com>, 2019
+# Alhamzah Alnufaili <inactive+Alhamzah97@transifex.com>, 2019
+# Natalia Berdnikov <nberdnikov@edx.org>, 2019
+# shefaa abu jabel <shefaa.aj@gmail.com>, 2019
+# Roy Zakka, 2019
+# Sahbi BG <sahbi.g@gmail.com>, 2019
+# 
+#, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2018-06-28 14:17+0430\n"
-"PO-Revision-Date: 2018-12-20 23:37+0500\n"
-"Language: ar\n"
+"POT-Creation-Date: 2019-07-27 17:07+0300\n"
+"PO-Revision-Date: 2019-07-27 13:51+0000\n"
+"Last-Translator: Sahbi BG <sahbi.g@gmail.com>, 2019\n"
+"Language-Team: Arabic (https://www.transifex.com/open-edx/teams/6205/ar/)\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=6; plural=(n==0 ? 0 : n==1 ? 1 : n==2 ? 2 : n%100>=3 "
-"&& n%100<=10 ? 3 : n%100>=11 && n%100<=99 ? 4 : 5);\n"
-"Last-Translator: \n"
-"Language-Team: \n"
-"X-Generator: Poedit 2.2\n"
+"Language: ar\n"
+"Plural-Forms: nplurals=6; plural=n==0 ? 0 : n==1 ? 1 : n==2 ? 2 : n%100>=3 && n%100<=10 ? 3 : n%100>=11 && n%100<=99 ? 4 : 5;\n"
 
-#: problem_builder/answer.py problem_builder/instructor_tool.py
+#: answer.py:146 instructor_tool.py:140
 msgid "Long Answer"
-msgstr "مخرل شرسصثق"
+msgstr ""
 
-#: problem_builder/answer.py problem_builder/mixins.py
+#: answer.py:150 mixins.py:135
 msgid "Question ID (name)"
-msgstr "السؤال هي (رشوث)"
```

Comment:
Thanks for asking @pomegranited. I've read the translations in Arabic and it appears to be incorrectly translated -- because I'm Arabic :man_shrugging: 

It's possible that another language like Urdu or Persian have been used under the code `ar`.

@xitij2000 @umarmughal824 what do you think?

Reply:
Sorry for the delayed response! 

Unfortunately I don't have any knowledge of Arabic. My main involvement in previous PRs was in testing and ensuring that the translations showed up correctly. While I do often do a rough check on Google Translate for random passages, I don't really trust it since I know how easily it can mess up in my "native" language Hindi. 

Would you be able to help with a more accurate translation here? Would an inaccurate translation be better here than a missing one?

Reply:
Thanks @xitij2000 for the review! Regarding the Arabic translation it's just pure random punches on the keyboard, that is incorrect. 

So I thought that there's no point of keeping it in a popular XBlock and confuse learners, instead English should be shown.

I would love to help with the Arabic, but I still have several issues to tackle in terms of XBlock i18n that is of higher priority.

However, I wouldn't be surprised to have it translated by the community on the next pull.

What do you think?

Reply:
@omarithawi Sure. In that case I think it might be better to have a missing translation than an incorrect/misleading one I guess. 

The creators of the original PR for adding these translations themselves might be interested in fixing this. Will see if I can get in touch with them and get a response in time for this PR itself.

Reply:
Thanks for contacting the original PR creators @xitij2000 . Any luck there?

Reply:
@pomegranited Nope :-(

Reply:
No worries @xitij2000 .. Since @OmarIthawi has fixed up the translation pulling and compiling, it should be easy now for someone to go into Transifex and update the Arabic strings, and submit a new PR here.

Reply:
Yup, that sounds fair enough!